docs/content.md

```diff
@@ -555,6 +556,44 @@ Alternatively, if you need to insert a mailing lists component in an erb file, y
 )%>
 ```
 
+### Green arrow link (CTA) component
+
+You can use the green arrow link (call to action) component to create a more prominent hyperlink call to action. You can use the component:
+
+* when it's the next logical step on users' journeys
+* more than once on a page
+
+The component should not:
+
+* be used on every page, if there isn't a suitable use for it
+* to direct to pages that are not useful for most people e.g. veterans, SKITT
+* to direct to external sites like UCAS or Teach First
+
+It takes the following parameters and both should be specified:
+
+* link_text
+* link_target
+
+```yaml
+---
+cta_arrow_link:
+  arrowtime:
+    link_target: "https://google.com"
+    link_text: "Hello world"cta_mailinglist:
+---
+
+# My page
+
+$arrowtime$
+
+```
+Alternatively, if you need to insert a mailing lists component in an erb file, you can call it like this:
+
+```yaml
+
+<%= render "content/shared/links_and_buttons/cta_circle_button_link", href: @link_target = "/events/about-get-into-teaching-events", cta_link_text: @link_text = "Get Into Teaching event" %>
```

Comment:
This should be removed as the partial version has been deprecated in favour of the component. The correct code would be:

```
 <%= render CallsToAction::ArrowLinkComponent.new(link_target: "/events/about-get-into-teaching-event", link_text: "Get Into Teaching event") %>
 ```